rpa-manager/modules/ROOT/pages/processoperations-deploymentmap.adoc

```diff
@@ -1,11 +1,22 @@
 = Deployment Map
 
-The deployment map shows you the current status of each productive configuration of all processes for which you are a team member.
+The deployment map shows the current status of each run configuration of all processes for which you are a team member.
 
-The interrelationships of processes, RPA Bots and Configurations are mapped by a network.
+The interrelationships of processes, RPA Bots, and configurations are mapped as a network.
 
+You can view and manage all your run configurations in one place:
+
+* Switch between test and production configurations.
+* Switch between hierarchical and star network views.
+* Filter the bots and configurations by status.
 * Switch directly to the detailed view of a process or a configuration.
-* Execute, pause, and stop configurations directly from here.
+* Execute test configurations.
+* Execute, pause, and stop production configurations.
+
+You must manually keep configuration changes in RPA Manager and Anypoint Exchange in sync. Configurations might become out of sync in the following situations:
+
+* Invokable run configurations deleted in RPA Manager might still exist in Anypoint Exchange, but they cannot cannot be run by the bot and do not appear in RPA Manager.
```

Comment:
Avoid passive voice:
...but the bot cannot run them so they do not appear in RPA Manager.